function-template/Dockerfile

```diff
@@ -1,11 +1,17 @@
 ARG IMAGE
 FROM ${IMAGE}
 
-ARG FUNCTION_SRC=function.txt
+ARG HANDLER
+ENV HANDLER=$HANDLER
 
-# FUNCTION_MODULE env var is set in the base-image
-COPY ${FUNCTION_SRC} ${FUNCTION_MODULE}
+COPY . .
 
-RUN echo "  (require '[func-server.main :as m]) (m/run-server function)" >> ${FUNCTION_MODULE}
+RUN echo -n ${HANDLER} > /tmp/handler && \
+    [ -z "${HANDLER}" ] && \
```

Comment:
It looks like this will return an error code of 1 if a handler is specified and fail.